### PR TITLE
Ethers to viem

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Replacing instances of Ether.js use with Viem

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1215

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./configs/nextjs.json",
   "compilerOptions": {
+    "target": "es2017",
```
Comment (kesar) [inline tsconfig.json:4]: why?

File: package.json [hunk 2]
```diff
@@ -29,11 +29,6 @@
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
     "@ensdomains/ens-avatar": "^0.1.6",
```
Comment (Emmanueldmlr) [inline package.json:31]: I think we don't need this again.

**Comment by Emmanueldmlr** [comment]
@kesar, we were not about to test the bridge page. I guess for now, you are the only that can help with that.

**Comment by kesar** [comment]
lets assume bridge works, and i will test it once its merged. anyhow a follow up future task could be simulate a way in testnet to test the bridge. also the one you mentioned about migrating to another testnet. if they are not open we should open those issues

**Review by Emmanueldmlr — APPROVED** [review]

**Pull request merged**